src/test/java/org/skriptlang/skript/test/tests/regression/HangingEntity_4871.java

```diff
@@ -0,0 +1,88 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package org.skriptlang.skript.test.tests.regression;
+
+import java.util.Optional;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.entity.EntityType;
+import org.bukkit.entity.ItemFrame;
+import org.bukkit.event.hanging.HangingBreakEvent;
+import org.bukkit.event.hanging.HangingBreakEvent.RemoveCause;
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import ch.njol.skript.test.runner.SkriptJUnitTest;
+
+public class HangingEntity_4871 extends SkriptJUnitTest {
```

Comment:
Do we use the issue ID in class name or PR ID? as we use the PR ID in test scripts so we should keep consistency.
Maybe we can use PR ID and add a comment with issue URL

Reply:
It's always the issue PR, regression uses issues PR.